### PR TITLE
Drivetrain

### PR DESCRIPTION
**Pull request by UnknownSilicon**
Drivetrain and a few  .gitignore updates

### CODE REVIEW

**Comment by simon-andrews** [comment]
What happened to the Ant stuff? :cry:

File: src/org/robockets/tshirtcannon/subsystems/ExampleSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (jakekinsella) [inline src/org/robockets/tshirtcannon/subsystems/ExampleSubsystem.java]: Remove Example Subsystem

File: src/org/robockets/tshirtcannon/commands/ExampleCommand.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (jakekinsella) [inline src/org/robockets/tshirtcannon/commands/ExampleCommand.java]: Remove example command

File: src/org/robockets/tshirtcannon/RobotMap.java [hunk 3]
```diff
@@ -1,13 +1,18 @@
 package org.robockets.tshirtcannon;
 
-/**
- * The RobotMap is a mapping from the ports sensors and actuators are wired into
- * to a variable name. This provides flexibility changing wiring, makes checking
- * the wiring easier and significantly reduces the number of magic numbers
- * floating around.
- */
 public class RobotMap {
+    // For example to map the left and right motors, you could define the
```
Comment (jakekinsella) [inline src/org/robockets/tshirtcannon/RobotMap.java:8]: Remove these comments

File: src/org/robockets/tshirtcannon/Robot.java [hunk 4]
```diff
@@ -15,6 +20,10 @@
  */
 public class Robot extends IterativeRobot {
 
+	public static final ExampleSubsystem exampleSubsystem = new ExampleSubsystem();
```
Comment (jakekinsella) [inline src/org/robockets/tshirtcannon/Robot.java]: Remove declaration for ExampleSubsystem

File: src/org/robockets/tshirtcannon/OI.java [hunk 5]
```diff
@@ -1,11 +1,40 @@
...
-import edu.wpi.first.wpilibj.buttons.Button;
+import edu.wpi.first.wpilibj.Joystick;
+
 
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
+    //// CREATING BUTTONS
```
Comment (jakekinsella) [inline src/org/robockets/tshirtcannon/OI.java]: Remove these comments

File: build.xml [hunk 6]
```diff
@@ -1,30 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jakekinsella) [inline build.xml:1]: Add ant stuff back in

File: build.properties [hunk 7]
```diff
@@ -1,4 +0,0 @@
-# Project specific information
```
Comment (jakekinsella) [inline build.properties:1]: Add this back in

**Comment by UnknownSilicon** [comment]
No more ant! I'll try to get unto work with gradle instead. :smile)

**Comment by UnknownSilicon** [comment]
I did that ^ wrong

**Comment by jakekinsella** [comment]
The ant files still must be included for those of us who don't want to use Gradle

**Comment by UnknownSilicon** [comment]
Yea, I just realized I didn't include those

**Comment by Math-ias** [comment]
We already have a gitignore in master.
Also remove all that metadata stuff from your history by rebasing and stuff.
Your commits need better grammar (capitalization) no "I thinks".
drive.java needs to be capitalized.

Basically rebase everything to be prettier.

File: src/org/robockets/tshirtcannon/commands/drive.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
+package org.robockets.tshirtcannon.commands;
+
+import org.robockets.tshirtcannon.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class drive extends Command {
```
Comment (Math-ias) [inline src/org/robockets/tshirtcannon/commands/drive.java:7]: drive should be capitalized

**Comment by Math-ias** [comment]
It also looks like Jared owns some of the commits I would rebase that.

**Pull request closed without merging**